source/main.cpp

```diff
@@ -252,18 +277,59 @@ Result DownloadTitle(std::string titleId, std::string encTitleKey, std::string t
 
     printf("Starting - %s\n", titleName.c_str());
 
+    // If in install mode, download/install the SEED entry
+    if (selected_mode == install_direct)
+    {
+        // Download and install the SEEDDB entry if install mode
+        // Code based on code from FBI: https://github.com/Steveice10/FBI/blob/master/source/core/util.c#L254
+        // Copyright (C) 2015 Steveice10
+        u8 seed[16];
+        static const char* regionStrings[] = {"JP", "US", "GB", "GB", "HK", "KR", "TW"};
+        u8 region = CFG_REGION_USA;
+        CFGU_GetSystemLanguage(&region);
```

Comment:
That's what I thought, but the country code passed in seems to be arbitrary, the titleid is what actually matters (And the country code has to be valid in general). I based this off the stuff in FBI, and so far I haven't seen complaints about that not working right for them :)